ngraph/core/src/op/add.cpp

```diff
@@ -53,7 +53,7 @@ shared_ptr<Node> ngraph::operator+(const Output<Node>& arg0, const Output<Node>&
     return make_shared<op::Add>(arg0, arg1);
 }
 
-namespace
+namespace add
```

Comment:
why unnamed namespaces don't work?

Reply:
Unity build merges several source files into single one. So you would get one single source file with several unnamed namespaces with the same symbols. It fails to compile. Thus I have to use different symbol names even for internal stuff.